dbms/src/Common/UTF8Helpers.h

```diff
@@ -28,6 +29,19 @@ namespace UTF8
 {
 static const UInt8 CONTINUATION_OCTET_MASK = 0b11000000u;
 static const UInt8 CONTINUATION_OCTET = 0b10000000u;
+static const UInt32 UNICODEMax = 0x0010FFFF; // Maximum valid Unicode code point.
+static const UInt32 UTF8Error = UNICODEMax + 1; // the "error" code
```

Comment:
UNICODEMax -> UNICODE_MAX
UTF8Error -> UTF8_ERROR

Reply:
Nice, changed it.

Reply:
still `UNICODE_Max`

Reply:
Yeah, I didn't notice the upper case, update it now.